setup.py

```diff
@@ -10,10 +10,12 @@
     history = history_file.read()
 
 requirements = [
+    "pip>=18.1"
     "numpy>=1.16.4",
     "torch>=0.4.1",
+    "Sphinx>=1.7.1"
     "matplotlib>=2.0",
-    "scikit-learn>=0.18, <0.20.0",
+    "scikit-learn>=0.18",
```

Comment:
apparently there is this bug on sklearn 0.20 ? Should we update all the minimal version on this list to be the exact same there is on the previous requirements.txt ? At least we know these one work. For example, sklearn can be >= 0.20.3 ? We can do the same for all the other packages.